scripts/gtk-bundle-from-msys2.sh

```diff
@@ -212,6 +212,10 @@ copy_grep_and_dependencies() {
 	if [ "${own_arch}" == "${ABI}" -o -z "${MSYS2_ABI_PATH}" ]; then
 		bin_dir="/usr/bin"
 	else
+		# TODO we could maybe download MSYS2 from http://www2.futureware.at/~nickoe/msys2-mirror/distrib/
+		# extract the relevant files, download "grep" from
+		# http://www2.futureware.at/~nickoe/msys2-mirror/msys/${ABI}/grep-2.XX-1-${ABI}.pkg.tar.xz
+		# and so automate this even more than requiring an existing separate MSYS installation
```

Comment:
Yeah, but usually we build for the target ABI i686 and on my system MSYS2 is x86_64. If I would use grep.exe from my MSYS2 shell, it would be x86_64 and so not usuable for users with a 32bit Windows system.

But yeah, maybe we can even pull the dependencies from Pacman instead. I will have a look at this post-1.29.